db/inst/import-try/helpers.species.R

```diff
@@ -0,0 +1,71 @@
+# Helper functions for discriminating species
+library(taxize)
+library(data.table)
+
+get.plants.sources <- function(){
+    require(taxize)
+    common.plants <- c("acer rubrum", "zea mays", "tsuga canadensis")
+    dat <- gnr_resolve(common.plants)
+    plants.sources <- unique(dat$data_source_title)
+    all.sources <- gnr_datasources()
+    source.ids <- all.sources[all.sources$title %in% plants.sources, "id"]
+    return(source.ids)
+}
+
+all.sources <- get.plants.sources()
+
+fix.species <- function(dat){
```

Comment:
this `fix.species` function looks promising ... but I don't see it used anywhere. Is this under "TODO"?